README.md

```diff
@@ -434,7 +434,7 @@ ensure that the Tesseract OCR application is installed and configured properly.
 1.  Ensure that the necessary packages are available on the operating system 
     installation.
     ```
-    Git - https://git-scm.com/download/
+    Git - https://git-scm.com/downloads/win
```

Comment:
Thank you for your contribution.

Could you please make the following change?

We want to include support for both GNU/Linux distributions and Microsoft Windows.

```suggestion
    Git - https://git-scm.com/downloads
```

Reply:
Great idea.
I added the appropriate links and some useful informations for the installation in GNU/Linux distributions and Microsoft Windows.